scripts/setup_chain.sh

```diff
@@ -18,6 +18,9 @@ OSMOSIS_CHAIN_LINK_DATA="$SCRIPT_DIR/osmosis-chain-link-data.json"
 
 desmos() {
 	"$SCRIPT_DIR/desmos" --home="$DESMOS_HOME" "$@"
+
+  # Wait tx including block
+  sleep 1
```

Comment:
Wait 1 second to wait for tx including a block since sync mode doesn't ensure it.